Solution1/Solution1.Tests/Constants.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Weather.Tests
+{
+    public static class Constants
```

Comment:
don't use in tests any global logic, constants and so on, tests should be undependent from each other